mlir/lib/Bindings/Python/IRCore.cpp

```diff
@@ -2989,8 +3014,7 @@ void mlir::python::populateIRCore(py::module &m) {
            py::arg("binary") = false, kOperationPrintStateDocstring)
       .def("print",
            py::overload_cast<std::optional<int64_t>, bool, bool, bool, bool,
-                             bool, py::object, bool>(
-               &PyOperationBase::print),
+                             bool, py::object, bool>(&PyOperationBase::print),
```

Comment:
nit: seems like unrelated format difference snuck in.

Reply:
i don't think there's anything that can be done about this? presumably @uenoku just did a `clang-format` and this happened? ie if this is reverted then `clang-format` will complain here as well.

Reply:
Yes, `Check code formatting` fails otherwise so I'll just keep it as is.

Reply:
Huh weird, I guess that snuck in via a previous change.